docs/atomic-storage.1.md

```diff
@@ -84,32 +89,39 @@ command will try to first move its data to some other device in the pool.
 Remove all block devices from the storage pool that are currently unused.
 
 **--driver**
-Backend storage driver for containers.  This options the storage driver.
-Drivers supported: devicemapper, overlay, overlay2
+Add the backend storage driver name to /etc/sysconfig/docker-storage-setup.
+Drivers supported: devicemapper, overlay, overlay2.
+This would set docker-storage-setup option: STORAGE_DRIVER.
 
 **--lvname**
-Logical volume name for container storage.
-E.g. --lvname="container-root-lv"
+Add the logical volume name for container storage to
+/etc/sysconfig/docker-storage-setup. E.g. --lvname="container-root-lv"
 Note: You must set --rootfs when setting --lvname.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_NAME.
 
 **--rootfs**
-Mountpath where logical volume for container storage would be mounted.
-E.g. --rootfs="/var/lib/containers". This will mount "container-root-lv"
-on "/var/lib/containers".
+Add the mountpath where logical volume for container storage would be mounted to
+/etc/sysconfig/docker-storage-setup. E.g. --rootfs="/var/lib/containers". This
+will mount "container-root-lv" on "/var/lib/containers".
 Note: You must set --lvname when setting --rootfs.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_MOUNT_PATH.
 
 **--lvsize**
-Logical volume size for container storage.
+Add logical volume size for container storage to /etc/sysconfig/docker-storage-setup.
 E.g. --lvsize="20%FREE". It defaults to 40% of all free space.
 --lvsize can take values acceptable to "lvcreate -L" as well
 as some values acceptable to "lvcreate -l". If user intends to pass
 values acceptable to "lvcreate -l", then only those values which
 contains "%" in syntax are acceptable.  If value does not contain
 "%" it is assumed value is suitable for "lvcreate -L".
 Note: You must set --lvname and --rootfs when setting --lvsize.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_SIZE.
```

Comment:
+E.g atomic storage modify --llvsize=20%FREE would add
 +CONTAINER_ROOT_LV_SIZE=20%FREE to /etc/sysconfig/docker-storage-setup.